### ISSUE TITLE
Homepage - Carousel in News and Featured Products: Missing padding in the right side.

### ISSUE THREAD

**Issue by maehtitan** [titan-cebu/Civil-and-General-Distributors#134]
Viewport size: 768px only

![image](https://cloud.githubusercontent.com/assets/13363508/15697057/14e0ca4c-27e9-11e6-9f1d-1c28cb1312e7.png)

**Comment by ganesh-titan** [comment]
@maehtitan : As per design (768px) we are implemented.  
![padding_both](https://cloud.githubusercontent.com/assets/13450883/15737623/aca57040-28c4-11e6-98a2-4435091f56cf.jpg)

**Comment by maehtitan** [comment]
@ganesh-titan I will give you a video and an image for the actual device, it doesn't have a padding both ends as per design. But you have implemented a padding in the left side, so I have reported that it isn't consistent with the right side that's why I have raised this as an issue. If you have followed the design correctly without padding both ends this will not raised in the first place. Either you remove the padding in the left side completely or have it consistent on both sides.


Video URL here: http://screencast.com/t/YoI34YkFWGJx
![image](https://cloud.githubusercontent.com/assets/13363508/15766217/c0adce16-296e-11e6-9a05-7da8fc97cccc.png)
